### PR TITLE
Create additional setup guides for top API sources

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
Creates additional setup guides for:
- Exchange Rates API
- Google Ads
- Google Sheets
- JIRA
- Slack
- Stripe
- Zendesk Support

Separating out the setup guides is a cleaner approach for the doc in comparison to OSS / Cloud tags, as it allows for the addition of the text at the bottom that links out to the original doc. 

Also does as hot-fix for Shopify, where the Streams information was duplicated inadvertendly

### CODE REVIEW

File: docs/integrations/sources/shopify.md [hunk 1]
```diff
@@ -125,40 +60,44 @@ Add the following scopes to your custom app to ensure Airbyte can sync all avail
...
+
+## Troubleshooting tips
+
+Check out common troubleshooting issues for the Shopify source connector on our Discourse [here](https://discuss.airbyte.io/tags/c/connector/11/source-shopify).
+
+## Supported Streams
 
 This Source is capable of syncing the following core Streams:
 
-* [Articles](https://shopify.dev/api/admin-rest/2022-01/resources/article)
```
Comment (sherifnada) [inline docs/integrations/sources/shopify.md:132]: how come you removed articles and blogs?

File: docs/integrations/sources/shopify.md [hunk 2]
```diff
@@ -5,77 +5,12 @@ description: >-
...
-
-The Shopify source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
-
-This source can sync data for the [Shopify REST API](https://shopify.dev/api/admin-rest) and the [Shopify GraphQl API](https://shopify.dev/api/admin-graphql).
-
-## Troubleshooting
-
-Check out common troubleshooting issues for the Shopify source connector on our Discourse [here](https://discuss.airbyte.io/tags/c/connector/11/source-shopify).
-
-### Output schema
```
Comment (sherifnada) [inline docs/integrations/sources/shopify.md:25]: why was this removed from here? Seems useful to have

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/shopify.md (hunk 1 above)
Comment (nataliekwong) [reply docs/integrations/sources/shopify.md:132]: Thanks for catching this. They were in one list but not the other, and I must have taken the older list. Added back.

File: docs/integrations/sources/shopify.md (hunk 2 above)
Comment (nataliekwong) [reply docs/integrations/sources/shopify.md:25]: It still exists! It was just duplicated twice so I removed the first instance of it. It is now located above the Supported Streams but under the Setup Guide.
